contrib/README.md

```diff
@@ -0,0 +1,368 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports ingress, such as GKE. If your environment doesn't have ingress support
+set up, please follow the [minikube](MINIKUBE.md) instructions to set this up for your development environment.  
+
+## Contents
+
+1. [Quickstart](#quickstart)
+    1. [Using helm](#using-helm)
+    1. [Using custom resources](#using-custom-resources)
+1. [Examples](#examples)
+    1. [Register a function](#register-a-function)
+    1. [Query all function](#query-all-function)
+    1. [Register an event](#register-an-event)
+    1. [Query all events](#query-all-events)
+    1. [Register a subscription](#register-a-subscription)
+    1. [Query all subscriptions](#query-all-subscriptions)
+1. [Configuration](#configuration)
+1. [Cleanup](#cleanup)
+
+## Quickstart
+
+Make sure you have helm installed on your machine and run `helm init` on your k8s cluster. This will set up the
+`helm` and `tiller` functions required for easy deployment of config files to your cluster. You can follow
+instructions [here](https://docs.helm.sh/using_helm/#quickstart) if you have not set this up previously.
+
+**NOTE:** This portion of the config expects you to have a pre-existing kubernetes cluster (not minikube). For 
+local development please check the [minikube](MINIKUBE.md) information below.
+
+Once installed, navigate to the `event-gateway/contrib/helm` folder and install the following components:
+
+**etcd-operator**
+```bash
+helm install stable/etcd-operator --name ego [--namespace <namespace>]
```

Comment:
Is the release name (--name) required? I guess its autogenerated if not specified

Reply:
Yes, that's correct. It's better to name it than to go searching for the service later.

```
[sborza@icebox]:~/src/github.com/serverless/event-gateway (k8s):$ helm install -h
This command installs a chart archive.

  ...

Usage:
  helm install [CHART] [flags]

Flags:
      --ca-file string           verify certificates of HTTPS-enabled servers using this CA bundle
      --cert-file string         identify HTTPS client using this SSL certificate file
      --dep-up                   run helm dependency update before installing the chart
      --devel                    use development versions, too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored.
      --dry-run                  simulate an install
      --key-file string          identify HTTPS client using this SSL key file
      --keyring string           location of public keys used for verification (default "/home/sborza/.gnupg/pubring.gpg")
  -n, --name string              release name. If unspecified, it will autogenerate one for you
  ...
```